### ISSUE TITLE
Facebook login not working

### ISSUE THREAD

**Issue by Jithinag007** [facebook/react-native-fbsdk#459]
New version xcode facebook login not working

**Comment by ayouidaniel** [comment]
+1

**Comment by aybarsyalcin** [comment]
+1

**Comment by nonotest** [comment]
anyone getting GraphQLHttpFailureDomain(401) on android now ? only when testing from a real device...

**Comment by dmamaril** [comment]
+1

**Comment by ayouidaniel** [comment]
> @Jithinag007 is your issue like this? https://i.stack.imgur.com/BpsHY.png

No, that seems something related to react-native linking or native import (module missing).
My problem was related to RTCLinking that is unable to resolve Facebook deep linking

**Comment by tuledev** [comment]
https://github.com/facebook/react-native-fbsdk/issues/549#issuecomment-503525279

**Comment by bernhardt1** [comment]
were y'all able to upgrade to the latest version and get login working?
